### ISSUE TITLE
Can anyone help me please? I cannot get the expected results.

### ISSUE THREAD

**Issue by robin0506** [emilwallner/Screenshot-to-code-in-Keras#29]
I clone the project from github to local and run the HTML version in local environment without changing any code. However, I don't get the expected results. The result is like,  
<img width="1034" alt="wx20180626-160501 2x" src="https://user-images.githubusercontent.com/18025979/41897738-b6b73666-795a-11e8-9249-9830bfeeb443.png">

**Comment by emilwallner** [comment]
Increase the epoch value here:
model.fit([image_data, X], y, batch_size=64, shuffle=False, epochs=2)
